src/content/utils.ts

```diff
@@ -283,3 +283,13 @@ const getTermFromSectionDetailsString = (
   if (includes2024) return Term.winterOne
   return Term.winterTwo
 }
+
+export const filterSectionsByWorklist = (sections: ISectionData[], worklist: number): ISectionData[] => {
+  const sectionsForWorklist:ISectionData[] = []
+  for (const section of sections) {
```

Comment:
this is a nitpick, but you could've probably used `sections.filter` here. not a really necessary change though.